spec/rubocop/cop/style/block_delimiters_spec.rb

```diff
@@ -490,6 +490,36 @@
       RUBY
     end
 
+    it 'allows when :[] is chained' do
+      expect_no_offenses(<<-RUBY.strip_indent)
+        foo = [{foo: :bar}].find { |h|
+          h.key?(:foo)
+        }[:foo]
+      RUBY
+      expect_no_offenses(<<-RUBY.strip_indent)
```

Comment:
This example doesn't match the `it` description. In general I'd have one it per example.

Reply:
👌